src/ext/java/org/opentripplanner/ext/transmodelapi/model/timetable/DatedServiceJourneyQuery.java

```diff
@@ -33,5 +42,90 @@ public static GraphQLFieldDefinition createGetById(
                 .build();
     }
 
+    public static GraphQLFieldDefinition createQuery(
```

Comment:
We spoke about this briefly on slack but I'll put it here as well for traceability. This query can be very costly from a performance point of view and might do well with some form of filtering to limit what's contained in the response (maybe by adding a service date argument?).